docs/.vitepress/theme/components/icons/IconInfo.vue

```diff
@@ -32,12 +32,28 @@ const DiamondIcon = createLucideIcon('Diamond', diamond)
       <IconDetailName class="icon-name">
         {{ icon.name }}
       </IconDetailName>
-      <div v-if="icon.externalLibrary" class="icon-external-lib">
-        <DiamondIcon fill="currentColor" :size="12"/>
+      <div
+        v-if="icon.externalLibrary"
+        class="icon-external-lib"
+      >
+        <DiamondIcon
+          fill="currentColor"
+          :size="12"
+        />
         {{ icon.externalLibrary }}
       </div>
+      <Badge
+        v-if="icon.deprecated"
+        class="deprecated-badge"
+        :title="`Reason: ${icon.deprecationReason}. To be removed in ${icon.toBeRemovedInVersion}`"
```

Comment:
```suggestion
        :title="deprecationReasonTemplate(…"
```

https://github.com/lucide-icons/lucide/blob/f238fe1b1858d4fbf6b24b1ef89d6822969712ea/tools/build-icons/utils/deprecationReasonTemplate.mjs#L1-L21

Reply:
thanks for your suggestion. I will close this PR and open a new one because there is some erros in commits that a need to fix.